src/vs/workbench/contrib/terminal/browser/links/terminalWordLinkDetector.ts

```diff
@@ -0,0 +1,92 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IConfigurationService } from 'vs/platform/configuration/common/configuration';
+import { ITerminalSimpleLink, ITerminalLinkDetector, TerminalLinkType } from 'vs/workbench/contrib/terminal/browser/links/links';
+import { convertLinkRangeToBuffer, getXtermLineContent } from 'vs/workbench/contrib/terminal/browser/links/terminalLinkHelpers';
+import { ITerminalConfiguration, TERMINAL_CONFIG_SECTION } from 'vs/workbench/contrib/terminal/common/terminal';
+import { IBufferLine, Terminal } from 'xterm';
+
+const enum Constants {
+	/**
+	 * The max line length to try extract word links from.
+	 */
+	MaxLineLength = 2000
+}
```

Comment:
```suggestion
//The max line length to try to extract word links from.
const MaxLineLength = 2000;
```

Reply:
Consider the output of the examples below:

https://www.typescriptlang.org/play?#code/MYewdgzgLgBApmArgWxgYXNAhmKEYDeAUDDALJYAeAMggOZQAWMAvDAEwAM3RAvkUVCRYyKrTANG7Vh26cA3EA

https://www.typescriptlang.org/play?#code/MYewdgzgLgBApmArgWxgYXNAhmKEYDeAUDDALJYAeAMggOZQAWMAvDAEwAM3RAvkUVCQQAGzgA6ESDoAKAOSIIcGODhyANOkxQcecRRr0mASgDcAodBjIqtMA0btWHbp3ODMoiVNkKlMKAB3EA1rWyNHMyA